File: Ohm web/sol.js
function sol(net) {
	const serie = (...args) => args.reduce((acc,cur) => acc + cur, 0);
	const parallel = (...args) => 1 / (args.reduce((acc, cur) => acc + ((cur == 0) ? 0 : 1/cur), 0));
	const burned = (...args) => 0;

	return eval(net.replace(/\(/g, 'serie(').replace(/\[/g, 'parallel(').replace(/\{/g, 'burned(').replace(/\]/g, ')').replace(/\}/g, ')'));
}

//result
let rl = require('readline').createInterface({
    input: process.stdin
});

rl.on("line", (resistances) => {
	rl.close();
	console.log(sol(resistances).toFixed(2));
});

File: Electro equivalence/sol.py
def sol(net):
    def serie(*args):
        return sum(args)
    def parallel(*args):
        return 1 / sum((0 if (cur == 0) else (1/cur)) for cur in args)
    def burned(*args):
        return 0

    return eval(net.replace('(', 'serie(').replace('[', 'parallel(').replace('{', 'burned(').replace(']', ')').replace('}', ')'))

# result
resistances = input("")
print("{:.2f}".format(sol(resistances)))